data/campaigns/Eastern_Invasion/utils/abilities.cfg

```diff
@@ -536,20 +549,28 @@ When an enemy unit of lower level within a 2 hex radius engages in combat, its a
         description= _ "This attack combines the arcane type with the type of weapon used so that resistance to the arcane type does not penalize the user."
         special_note=_ "This unit can use the arcane type when the opponent is particularly sensitive to it in relation to the weapon on which it is applied."
         #textdomain wesnoth-ei
-        alternative_type=arcane
-        active_on=offense
-        [filter_opponent]
-            [not]
-                [experimental_filter_ability_active]
-                    tag_name=resistance
-                    id=shroud
-                [/experimental_filter_ability_active]
-            [/not]
-        [/filter_opponent]
+        {COMMON_ARCANE}
     [/damage_type]
     [damage_type]
         id=arcane_damage_defense
         alternative_type=arcane
         active_on=defense
     [/damage_type]
 #enddef
+
+#define ABILITY_ARCANE_EI
+    [damage_type]
+        id=arcane_blessing
+        name= _ "arcane blessing"
+        description= _ "All attacks combines the arcane type with the type of weapon used so that resistance to the arcane type does not penalize the user."
```

Comment:
We can't add untranslated strings to 1.18, even though the existing strings wouldn't fit this either.

Reply:
naame and description removed